app/src/main/AndroidManifest.xml

```diff
@@ -8,6 +8,7 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
+	    android:usesCleartextTraffic="true"
```

Comment:
Please check domain based clear text traffic.

Reply:
This line will be added by developers in their app manifest if needed.